src/main/scala/amailp/intellij/robot/psi/reference/ResourceValueReference.scala

```diff
@@ -8,6 +8,8 @@ import com.intellij.openapi.vfs.VirtualFile
 import com.intellij.psi.{PsiElement, PsiReferenceBase}
 
 class ResourceValueReference(element: ResourceValue) extends PsiReferenceBase[ResourceValue](element) with ExtRobotPsiUtils {
+  val resourceFilePath: String = getElement.getText.replace("${/}", "/")
```

Comment:
I am not 100% sure that subsequent calls to a getElement would result in the same element. Often Idea reuses the same objects even when the text changes. For this reason I would not use here a `val` rather a `def`.
```suggestion
  def resourceFilePath: String = getElement.getText.replace("${/}", "/")
```